server/database/migrations/20210619173759_create-posts_activity-table.js

```diff
@@ -0,0 +1,50 @@
+// utils
+const logger = require("../../utils/logger");
+
+exports.up = (knex) => {
+	return knex.schema
+		.createTable("posts_activity", (table) => {
+			table.uuid("id").notNullable().primary();
+			table.string("type", 50).notNullable();
+			table.uuid("posts_comments_id").references("id").inTable("posts_comments");
```

Comment:
Codacy has a fix for the issue: [Replace `.uuid("posts_comments_id").references("id")` with `⏎↹↹↹↹.uuid("posts_comments_id")⏎↹↹↹↹.references("id")⏎↹↹↹↹`](https://app.codacy.com/gh/logchimp/logchimp/pullRequest?prid=7771592)
```suggestion
			table
```